functions/get-team.ts

```diff
@@ -11,8 +11,22 @@ exports.handler = async (event) => {
       headers: { Allow: "GET" },
     };
   }
+
+  function isDangerousHandle(s) {
+    return s.match(/^[0-9a-zA-Z_\-]+$/) === null;
```

Comment:
Seems that in line https://github.com/code-423n4/code423n4.com/pull/2330/files#diff-f56b9b0742f3b1b01343dbf9754332291dcf6d5c81644f6fd6c5af840d9563deR35 it will require the ".json"

Reply:
Yes, you are right

Reply:
Yes, you're right, I mixed up the names with the phone.